inc/saiport.h

```diff
@@ -1713,6 +1713,28 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_AUTO_NEG_FEC_MODE_OVERRIDE,
 
+    /**
+     * @brief Low latency variation
+     *
+     * False: Disable Low latency
+     * True: Enable Low latency
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_LOW_LATENCY_VARIATION,
+
+    /**
+     * @brief Query the Auto Negotiated Resolved FEC
+     *
+     * Auto negotiated FEC status is applicable on Auto Negotiation good state
+     *
+     * @type sai_s32_list_t sai_port_fec_mode_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_AUTO_NEG_FEC_MODE,
```

Comment:
As discussed previously, we intend to add an attribute to query the operational FEC mode. This mode could have been set manually, or autonegotiated, or set by the driver/firmware. Once we add this attribute, it would seem that `SAI_PORT_ATTR_AUTO_NEG_FEC_MODE` is redundant. Do you agree?

Reply:
This attribute will be used to query the resolved AN FEC after AN completion to AN Good state.  Port operational  FEC mode and Resolved FEC mode can be same or different in certain chips. We can not use operational  FEC mode to query the resolved auto negotiated FEC mode.

Reply:
SAI_PORT_ATTR_FEC_MODE attribute is used to configure port FEC. SAI_PORT_ATTR_ADVERTISED_FEC_MODE is used to set FEC when AN is enabled. SAI_PORT_ATTR_AUTO_NEG_FEC_MODE is the resolved AN FEC after AN completion. Resolved/advertised/configured FEC values can be different.

Reply:
I understand that resolved/advertised/configured FEC values can be different. However, I don't understand how the resolved FEC mode can be different from the operational FEC mode if AN is enabled. Would you mind discussing this in the next community meeting, please?

Reply:
Per our discussion in the meeting, could you please add a document describing the scenario(s) where `SAI_PORT_ATTR_AUTO_NEG_FEC_MODE` will be used?